document/json.go

```diff
@@ -35,6 +35,17 @@ func (v Value) MarshalJSON() ([]byte, error) {
 		x = string(v.V.(string))
 	case DurationValue:
 		x = v.V.(time.Duration).String()
+	case DoubleValue:
```

Comment:
This is a good catch, but this was already fixed in another PR, you should ignore this for the moment